### PR TITLE
data: dev-rakeshshaw

### PR DESCRIPTION
**Pull request by dev-rakeshshaw**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4818"></a>

### CODE REVIEW

File: data/dev-rakeshshaw.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Rakesh Shaw",
+  "type": "personal",
+  "bio": "👋Upcoming **Dev**👋. Success requires persistent practice, guided by patience."
+  "avatar": "https://github.com/dev-rakeshshaw.png"
```
Comment (sidharthviz) [inline data/dev-rakeshshaw.json:5]: Hi @dev-rakeshshaw There' no need to add the Avatar,  You can remove it.
 Thank you

Comment (sidharthviz) [inline data/dev-rakeshshaw.json:5]: Hi @dev-rakeshshaw  You can remove the avatar, There's no need to add it.
rest looks good 👍🏼

**Review by sidharthviz — CHANGES_REQUESTED** [review]
you can remove the avatar

**Comment by shubhsharma19** [comment]
LGTM

File: data/dev-rakeshshaw.json (hunk 1 above)
Comment (dev-rakeshshaw) [reply data/dev-rakeshshaw.json:5]: Can you please guide me on how can I remove the avatar?
Actually, I am new to this type of thing.

Comment (priyankarpal) [reply data/dev-rakeshshaw.json:5]: > Can you please guide me on how can I remove the avatar? Actually, I am new to this type of thing.

hey @dev-rakeshshaw just go `File Changed` tab 
Screenshot from 2023-02-14 19-38-50
Click on dot menu > edit file & remove the line
Screenshot from 2023-02-14 19-39-34

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/dev-rakeshshaw.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/dev-rakeshshaw",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Chat with me on Telegram",
+      "url": "https://t.me/dev_rakeshshaw",
+      "icon": "FaTelegram"
+    },
```
Comment (priyankarpal) [inline data/dev-rakeshshaw.json:21]: Lastly delete the comma.
```suggestion
    }
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/dev-rakeshshaw.json (hunk 1 above)
Comment (dev-rakeshshaw) [reply data/dev-rakeshshaw.json]: Thanks, brother.

File: data/dev-rakeshshaw.json (hunk 2 above)
Comment (dev-rakeshshaw) [reply data/dev-rakeshshaw.json]: Thanks, brother

**Review by priyankarpal — DISMISSED** [review]
Now it's looking good. 🎉

**Review by Hrishi5111998 — DISMISSED** [review]

File: data/dev-rakeshshaw.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Rakesh Shaw",
+  "type": "personal",
+  "bio": "👋Upcoming **Dev**👋. Success requires persistent practice, guided by patience."
```
Comment (kumarsonsoff3) [inline data/dev-rakeshshaw.json:4]: Please add a missing comma `,` here.

```suggestion
  "bio": "👋Upcoming **Dev**👋. Success requires persistent practice, guided by patience.",
```

**Pro Tip:** You can commit the requested changes directly using the <kbd>Commit suggestion</kbd> button above.

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by shubhsharma19 — APPROVED** [review]
Looks good!

File: data/dev-rakeshshaw.json (hunk 3 above)
Comment (dev-rakeshshaw) [reply data/dev-rakeshshaw.json]: Thanks for the suggestion, Bro.

**Review by priyankarpal — APPROVED** [review]
Awesome 🎉

**Review by Krishnapro — APPROVED** [review]
Now looks good to me

**Comment by dev-rakeshshaw** [comment]
> Now looks good to me

thanks bro❤

**Comment by dev-rakeshshaw** [comment]
I will com back soon.

**Comment by shubhsharma19** [comment]
> I will com back soon.

Is there an issue?

**Pull request closed without merging**